add_npm_package.sh

```diff
@@ -25,31 +26,47 @@ if !(program_exists npm2rpm); then
   exit 1
 fi
 
+if [[ -z $VERSION ]] ; then
+  if !(program_exists curl) ;  then
+    echo "curl is not installed - you can install it with:"
+    echo "sudo yum install curl"
+    exit 1
+  fi
+
+  if !(program_exists jq) ;  then
+    echo "jq is not installed - you can install it with:"
+    echo "sudo yum install jq"
+    exit 1
+  fi
+
+  VERSION=$(curl -s https://api.npms.io/v2/package/$NPM_MODULE_NAME | jq -r .collected.metadata.version)
```

Comment:
Would be cool if there was a way to default to the latest version allowed by package.json, but I am fine with this for now as a default.

Reply:
I figured that if a dependency is added it's usually the latest so this is good enough for now.